apps/commercetools/src/AppConfig/index.spec.tsx

```diff
@@ -70,8 +70,8 @@ describe("AppConfig", () => {
       [/commercetools Project Key/, ""],
       [/Client ID/, ""],
       [/Client Secret/, ""],
-      [/^API Endpoint/, ""],
```

Comment:
you've got `singleQuote: true` in prettierrc above, why wasn't this fixed?

Reply:
I didn't run prettier on this file.. prettier is messed up all over the new apps repo